selftests/__init__.py

```diff
@@ -2,9 +2,34 @@
 import sys
 import unittest
 
+try:
+    from unittest import mock
+except ImportError:
+    import mock
+
+
 #: The base directory for the avocado source tree
 BASEDIR = os.path.abspath(os.path.join(os.path.dirname(os.path.abspath(__file__)), '..'))
 
 #: The name of the avocado test runner entry point
 AVOCADO = os.environ.get("UNITTEST_AVOCADO_CMD",
                          "%s ./scripts/avocado" % sys.executable)
+
+
+def recent_mock():
```

Comment:
Could we keep this change on #2848 (or here) only?

Reply:
A merge will take care of this, only the first commit merged will make it.